### ISSUE TITLE
Quick Switch: Arrow Keys Do Not Work

### ISSUE THREAD

**Issue by ekristen** [rancher/dashboard#5718]
**Setup**
- Rancher version: 2.6.3
- Browser type & version: Firefox / 99.0.1

**Describe the bug**

When using the quick switch I would expect the arrow keys to work to allow navigating through the resources found after typing a query, unfortunately I have to use a mouse to select the entry instead.

**To Reproduce**

Open up rancher, use the quick switch.

**Result**

You must use a mouse to select the resource you want instead of being able to directly search and or navigate using the arrow keys followed by the enter key.

**Expected Result**

Open up rancher, use the quick switch, the first item should be highlighted by default, and arrows keys should allow going through the list one by one, if I type anything the list should change and the first item should be hightlighted again and the arrow keys should allow me to key down followed by the enter key to select.

**Screenshots**

**Additional context**

**Detailed Description**

**Context**

**Comment by aalves08** [comment]
Won't fix - covered by new accessibility issues
